ArduPlane/Parameters.cpp

```diff
@@ -1274,6 +1274,15 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     AP_SUBGROUPINFO(precland, "PLND_", 35, ParametersG2, AC_PrecLand),
 #endif
 
+#if AP_RANGEFINDER_ENABLED
+    // @Param: RNGFND_LND_ORNT
```

Comment:
Do we need this on normal planes or could it be a quadplane param? I guess the custom orientations are really the only use case? We could gate on `AP_CUSTOMROTATIONS_ENABLED` although its enabled on most everything.

Reply:
we only disable custom rotations on periphs, so it doesn't win us anything
I think we want the orientation to be settable on normal planes too, you may want one which is pointing fwd 45 degrees